pkg/gateway/gateway.go

```diff
@@ -533,6 +535,12 @@ func (n *jfsObjects) CopyObject(ctx context.Context, srcBucket, srcObject, dstBu
 	}
 	dst := n.path(dstBucket, dstObject)
 	src := n.path(srcBucket, srcObject)
+
```

Comment:
It should be handled like an object-tag, first set to the tmp object.

Reply:
Done.